common/utils/tests/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
 
-project(cgautilstests)
+set(TARGET_NAME cgautilstests)
```

Comment:
why do you define variable for this?

Reply:
It's a future proofing habit I have for cmake. These cmakelist files tend to grow, and it's sets the stage for how to address the target

Reply:
I fail to see how a TARGET_NAME variable helps.
Instead of
```
add_executable(cgautilstest)
target_link_libraries(cgautilstest ...)
somemorestuff(cgautilstest ...)
```
you write
```
add_executable(${TARGET_NAME})
target_link_libraries(${TARGET_NAME} ...)
somemorestuff(${TARGET_NAME} ...)
```
This can never be a list, so why is this more future proof?

Reply:
If you add more targets, you will have to introduce a second TARGET_NAME variable or overwrite the first one, which will only make it more obscure.

The only benefit is that you can do c&p from CMakeLists.txt from other similar test directories.

Reply:
1. It puts the main target of the CMake file up top, making it more readable.
2. Creates a template for future files of the same type. New test CMake files can use this as a template and only replace target_name and sources.
3. Low likelihood (but has happened before in drive works e.g.) when copying string names multiple times can easily make mistake with similarly named targets.

We can dig into each of these if you'd like, but I see diminishing returns...

Reply:
We could. :) But I guess you are right it is beside the point. It is a detail after all.